File: 2.2P Counter/Counter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography;
using System.Text;
using System.Threading.Tasks;

namespace Counter
{
    public class Counter
    {
        private string _name;
        private int _count;

        public string Name //property
        {
            get 
            { 
                return _name; 
            }
            set 
            {
                _name = value; 
            }
        }

        public int Ticks
        {
            get
            {
                return _count;
            }
        }

        public Counter(string name)
        {
            _name = name;
            _count = 0;
        }

        public void Increment()
        {
            _count++;
        }

        public void Reset()
        {
             _count = 0;
        }

       

    }
}

File: 11.1P Clock in Python/ClockInPy/Counter.py
class Counter:
    
    def __init__(self, name):  #Constructor method for initializing a Counter object
        self.name = name  #Initialize the name of the counter
        self.count = 0  #Initialize the count of the counter to 0

    def increment(self):  #increment the count of the counter by 1
        self.count += 1

    def reset(self):  #reset the count of the counter to 0
        self.count = 0